### ISSUE TITLE
Version 1.0.0 breaks px-app-nav animation

### ISSUE THREAD

**Issue by gstroup** [PredixDev/px-vis-timeseries#44]
## Expected behavior and actual behavior:
App nav expand/collapse behavior should work in an app with px-vis-timeseries.  Version 0.8.7 of this component works nicely with px-app-nav.  The new version does not.

## Steps to reproduce the problem:
Import px-vis-timeseries into an app with px-app-nav.  Click the hamburger to expand / collapse.  Menu does not collapse.  Error displays in debug console.

```
Uncaught TypeError: Cannot read property 'cancel' of null
    at b.Animation (web-animations-next-animation.js:41)
    at b.AnimationTimeline._play (timeline.js:48)
    at HTMLDivElement.Element.animate (keyframe-effect-constructor.js:116)
    at Object.b.newUnderlyingAnimationForKeyframeEffect (keyframe-effect-constructor.js:134)
    at b.Animation._rebuildUnderlyingAnimation (web-animations-next-animation.js:90)
    at b.Animation (web-animations-next-animation.js:39)
    at b.AnimationTimeline._play (timeline.js:48)
    at Object.b.newUnderlyingAnimationForGroup (group-constructors.js:190)
    at b.Animation._rebuildUnderlyingAnimation (web-animations-next-animation.js:94)
    at b.Animation (web-animations-next-animation.js:39)
```

## Environment (_component version number, Browser (including version), operating system, hardware, etc_):
Chrome 56, Mac OS X Sierra
px-vis-timeseries 1.0.0
px-app-nav 1.10.7

## Code examples help us better understand the issue 

http://codepen.io/anon/pen/wJrJLB?editors=1001

Remove the import of px-vis-timeseries, and px-app-nav starts working again.

**Comment by nonmetalhail** [comment]
To clarify, you are not even loading a chart on the page, just the chart import. Correct?

**Comment by benoitjchevalier** [comment]
Interesting. App-nav and vis seem to load the same version of web-animations-js, wonder what's happening

**Comment by gstroup** [comment]
Yes, just the import will cause the problem.  I did load a chart on the page, then started removing things to isolate the problem, and found that just the import would cause the problem.
